blogposts/gitlab-adds-code-navigation-and-code-intelligence-by-sourcegraph.md

```diff
@@ -0,0 +1,90 @@
+
+<p class="container">
+  <div style="padding:56.25% 0 0 0;position:relative;">
+    <iframe src="https://player.vimeo.com/video/372226334?color=0CB6F4&amp;title=0&amp;byline=" style="position:absolute;top:0;left:0;width:100%;height:100%;" frameborder="0" webkitallowfullscreen="" mozallowfullscreen="" allowfullscreen=""></iframe>
+  </div>
+  <p style="text-align: center"><a href="https://vimeo.com/372226334" target="_blank">View on Vimeo</a></p>
+</p>
+
+## Built-in IDE-like features improve code reviews
+
+Code reviews become higher quality when developers have more context about the code they are reviewing. With this new integration, developers can traverse the code they are reviewing in the browser, investigate the impact of code changes thoroughly by finding all affected references and complete their code reviews faster.
+
+Higher-quality code reviews mean fewer bugs, and more time spent on building new features. Making open-source code easier to read and understand means faster identification of vulnerabilities, a better understanding of libraries, and less duplicated code. 
+
+Together, GitLab and Sourcegraph are creating a better world for enterprise and open source developers, consumers, and organizations.
+
+## When you can opt-in
+
+Sourcegraph code intelligence and code navigation will be available on November 20th for GitLab.com and November 30th for enterprise instances.
+
+## FAQ
+**What does this mean for the future of the browser extension?**
+<br>
+Until now, to get these features integrated seamlessly into their code host, users needed to install our popular [Sourcegraph for Chrome/Firefox extensions](https://docs.sourcegraph.com/integration/browser_extension) or install server-side extensions.
+The browser extension still exists and supports GitHub, GitLab, Bitbucket server, Phabricator, etc. This new integration just makes it so all GitLab users can get code navigation and code intelligence on GitLab without needing to install anything.
+
+**When will other code hosts get this treatment?**
+<br>
+Code navigation and code intelligence are essential, so we are working with [other major code hosts](https://docs.sourcegraph.com/integration) to incorporate these features using [open standards](https://docs.sourcegraph.com/integration).
+
+**How does this work for public and private code?** 
+<br>
+Soon all public code on GitLab.com will have Sourcegraph code navigation and code intelligence (for users who opt-in). Users of private code on GitLab.com or of self-managed GitLab instances can opt-in and configure GitLab to point to a self-managed Sourcegraph instance.
+
+**What languages does this support?**
+<br>
+The integration supports the [25 most used programming languages](https://sourcegraph.com/extensions?query=category%3A%22Programming+languages%22):<br>
+Clojure<br>
```

Comment:
This needs to be an unordered list.

Ideally, we use the images from the [Sourcegraph language extensions list](https://sourcegraph.com/extensions?query=category%3A%22Programming+languages%22) and display them as tiles to make this more visually appealing.

Reply:
@ryan-blunden If we only use images it defeats the purpose of SEO. Should I switch this file to a `tsx` file since `md` is a bit restrictive with lists?

Reply:
I don't think the languages in and of themselves will add significant SEO value here, but you could add the language text to a span with a class of "sr-only", or at least alt text to each image. Would just look a lot nicer.

> Should I switch this file to a tsx file since md is a bit restrictive with lists?

You just mark this up as a list using the `- ` syntax, and if you were going to do the language tiles, you can insert the raw HTML. It's great to strive to only use Markup syntax, but it's designed to be flexible by accepting HTML when needed.